docs/resources/samples/report-day-to-day-changes.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Record day-to-day changes in a workbook and report them with a flow
+description: Learn how to use Office Scripts and Power Automate to track value changes in a workbook
+ms.date: 08/25/2022
+ms.localizationpriority: medium
+---
+
+# Record day-to-day changes in a workbook and report them with a flow
```

Comment:
Suggest you use SEO terms in H1. Something like "Build a PowerAutomate flow to log and report changes in Excel"

Reply:
Is it okay for the title and H1 to not match?

Reply:
Yes, they CAN match (a lot of people do this) or use a different, but effective H1. So, match/not match. Both are OK as long as there's SEO value added!